### ISSUE TITLE
Remove `watchOptions` prop

### ISSUE THREAD

**Issue by WofWca** [mercs600/vue2-perfect-scrollbar#33]
And just always watch `options` instead.

1. The API is unintuitive, and unnecessary users can just create a non-reactive options object by themselves.
2. This prop also almost doubles the size of the package (if we don't count `perfect-scrollbar` itself

**Comment by mercs600** [comment]
Hi, thank you for proposition. I'm wondering what gives you reactive options when you have to reinitialize perfect scrollbar instance again ? PS instance is not reactive itself, so I have to destroy and create new one. What do you think ?

**Comment by WofWca** [comment]
> I'm wondering what gives you reactive options when you have to reinitialize perfect scrollbar instance again ?

Sorry, could you rephrase?

**Comment by mercs600** [comment]
Ok never mind, I did not understand you the first time -)  So you want it to watch options by default ?

**Comment by WofWca** [comment]
Yes.

**Comment by mercs600** [comment]
I will take into account, because it will be breaking change
